### ISSUE TITLE
what am i doing wrong?

### ISSUE THREAD

**Issue by max4all** [bitwalker/distillery#231]
Not so complicated Phoenix app developed on Mac and transferred to Ubuntu 16. Does this error make any sense? Thanks for help.

==> Release packaging failed due to errors:
    {{case_clause,
     {'EXIT',
         {function_clause,
             [{filename,join,[[]],[{file,"filename.erl"},{line,396}]},
              {erl_tar,split_filename,4,[{file,"erl_tar.erl"},{line,471}]},
              {erl_tar,create_header,3,[{file,"erl_tar.erl"},{line,400}]},
              {erl_tar,add1,4,[{file,"erl_tar.erl"},{line,323}]},
              {erl_tar,foreach_while_ok,2,[{file,"erl_tar.erl"},{line,1000}]},
              {systools_make,add_to_tar,3,
                  [{file,"systools_make.erl"},{line,1907}]},
              {systools_make,add_appl,7,
                  [{file,"systools_make.erl"},{line,1778}]},
              {systools_make,'-add_applications/5-fun-0-',6,
                  [{file,"systools_make.erl"},{line,1597}]}]}}},
 [{systools_make,'-add_applications/5-fun-0-',6,
      [{file,"systools_make.erl"},{line,1597}]},
  {lists,foldl,3,[{file,"lists.erl"},{line,1263}]},
  {systools_make,add_applications,5,[{file,"systools_make.erl"},{line,1596}]},
  {systools_make,mk_tar,6,[{file,"systools_make.erl"},{line,1590}]},
  {systools_make,mk_tar,5,[{file,"systools_make.erl"},{line,1566}]},
  {systools_make,make_tar,2,[{file,"systools_make.erl"},{line,339}]},
  {'Elixir.Mix.Releases.Archiver',make_tar,1,
      [{file,"lib/mix/lib/releases/archiver.ex"},{line,67}]},
  {'Elixir.Mix.Releases.Archiver',archive,1,
      [{file,"lib/mix/lib/releases/archiver.ex"},{line,17}]}]}

**Comment by bitwalker** [comment]
Yes, it means you have a module name which is longer than 100 characters when stored on disk with the `.beam` extension. This is a limitation of `erl_tar`, but one which I've fixed in the current master branch of OTP via a PR, which should be released with OTP 20 I hope. In the meantime, I would suggest either renaming the problem module, or if you aren't using hot upgrades/downgrades, you can pass `--no-tar` and tar up the `_build/<MIX_ENV>/rel/<app>` directory yourself.

**Comment by max4all** [comment]
Thanks @bitwalker
